monkey/monkey_island/cc/ui/src/components/configuration-components/ValidationFormats.js

```diff
@@ -8,22 +8,25 @@ const linuxAbsolutePathRegex = /^\// // path starts with `/`
 const linuxPathStartsWithEnvVariableRegex = /^\$/ // path starts with `$`
 const linuxPathStartsWithTildeRegex = /^~/ // path starts with `~`
 
-
 const windowsAbsolutePathRegex = /^([A-Za-z]:(\\|\/))/ // path starts like `C:\` OR `C:/`
 const windowsEnvVarNonNumeric = '[A-Za-z#\\$\'\\(\\)\\*\\+,\\-\\.\\?@\\[\\]_`\\{\\}~ ]'
 const windowsPathStartsWithEnvVariableRegex = new RegExp(
   `^%(${windowsEnvVarNonNumeric}+(${windowsEnvVarNonNumeric}|\\d)*)%`
 ) // path starts like `$` OR `%abc%`
 const windowsUncPathRegex = /^\\{2}/ // Path starts like `\\`
+
 const emptyRegex = /^$/
 
+const emailAddressRegex = new RegExp(/^[\S\-.]+@([\w-]+\.)+[\w-]{2,4}$/)
```

Comment:
Not totally sure about this. What all do we want to accept?

Reply:
Well, here's the regex from the RFC  http://www.ex-parrot.com/~pdw/Mail-RFC822-Address.html

Reply:
Seriously though, is there a library that does this validation? Maybe @ordabach can help?

Reply:
https://www.npmjs.com/package/email-validator ?

Reply:
Looks good to me. Going with this.